mongo/wrapper/helper.go

```diff
@@ -0,0 +1,9 @@
+package wrapper
+
+import (
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func MongoGetCollection(client *mongo.Client, dbName string, collectionName string) mongo.Collection {
+	return client.Database(dbName).Collection(collectionName)
```

Comment:
Do we really need this wrapper? IMO, I think the new driver API is cleaner than the open-source driver, and I don't see that it makes the code more compact.

Reply:
me making this PR was asking "do we need this?" :)

Reply:
I wanted to add GetAll here as well (wrapper around Find and cursor.All)

Reply:
I say let's not complicate this and use the driver natively